tool/tsh/common/dup2_linux_arm64.go

```diff
@@ -0,0 +1,34 @@
+// Teleport
+// Copyright (C) 2025 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+//go:build linux && arm64
```

Comment:
If dup3 is available on all supported linux platforms I'd rather use it on all linux rather than just linux/arm64.

Reply:
`dup3` is supported on linux on all architectures (since linux 2.6.27, which is much earlier than what go supports), `dup2` is liable to never get added to any new architectures, so the current fix will fail to compile as soon as we need to add support for a new architecture, whereas if we were to use dup3 for all linux builds the code would likely be fine.

Reply:
ok. i've done that. PTAL